File: 프로그래머스/2/12985. 예상 대진표/예상 대진표.java
import java.util.*;

class Solution
{
    public int solution(int n, int a, int b)
    {
        int answer = 0;
        double A = (double) a;
        double B = (double) b;
        while(A != B) {
            A = Math.ceil(A/2);
            B = Math.ceil(B/2);
            answer++;
        }
        return answer;
    }
}

File: 프로그래머스/2/12985. 예상 대진표/예상 대진표.kt
class Solution {
    fun solution(n: Int, a: Int, b: Int): Int {
        var x = a.toDouble()
        var y = b.toDouble()
        var res = 0
        
        while(x != y) {
            res+=1
            x = Math.ceil(x / 2)
            y = Math.ceil(y / 2)
        }
        
        return res
    }
}